### DISCUSSION TITLE
Do you use Alexa Fluor 750/DyLight 755 conjugates in your multiplexed imaging experiments?

### DISCUSSION THREAD

**Discussion by radtkea** [IBEXImagingCommunity/ibex_imaging_knowledge_base#58] [Polls]
We would like to know how many people use this channel for their imaging studies in hopes of encouraging more antibody suppliers to make direct conjugates. Thanks for your input!

**Comment by zmikulski** [comment]
We conjugate our abs with CF750 kits from Biotium, mostly because there are not enough commercially available conjugates. Don't know if this dye has the right properties for IBEX.

**Reply by radtkea** [reply]
Hi Z! Thanks for your comment. We haven't used CF750 yet. I suspect it will dye inactivate.

Glad you have arrived at the same conclusion as us: Great conjugate if you have the system and available reagents:)

Let's hope the suppliers will begin to offer more direct conjugates

Did you see: https://www.biolegend.com/en-us/ibex-applications/community
This is a great step in the right direction!

Thanks very much,
A
